tools.go

```diff
@@ -0,0 +1,10 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package hypersdk
+
+// TODO: Once we update to go1.24, we should register canoto as a tool in the
+// go.mod file.
+import (
+	_ "github.com/StephenButtolph/canoto/generate"
+)
```

Comment:
This allows us to inherit the canoto version from the `go.mod` file - rather than needing to specify the version in every file.